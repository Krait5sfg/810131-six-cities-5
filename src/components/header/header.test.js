import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header';
import {BrowserRouter} from 'react-router-dom';

it(`Header render correctly`, () => {
  const tree = renderer.create(<BrowserRouter><Header><React.Fragment /></Header></BrowserRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
