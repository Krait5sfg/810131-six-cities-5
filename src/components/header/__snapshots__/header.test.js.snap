// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render correctly 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      >
        <a
          className="header__logo-link"
          href="/"
          onClick={[Function]}
        >
          <img
            alt="6 cities logo"
            className="header__logo"
            height="41"
            src="img/logo.svg"
            width="81"
          />
        </a>
      </div>
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        />
      </nav>
    </div>
  </div>
</header>
`;
